### PR TITLE
Update: List updated with new participants

### PR DESCRIPTION
**Pull request by vimode**
Thanks for contributing to the [Hacktoberfest Swag List](https://hacktoberfestswaglist.com/) :smiley: :tada:! Before submitting your pull request, please check off as many of the items below as you can:

1. [x] I have read the [Contributing.md](https://github.com/crweiner/hacktoberfest-swag-list/blob/master/docs/contributing.md) file and formatted this PR correctly
2. [x] I'm not adding a company from the blocklist
3. [x] I make sure to fix things promptly if an error or suggestion comes up

Thanks and Happy Hacktoberfest! :tada:
Tagging @crweiner to take a look. :eyes:

Happy Hacktoberfest!!! 🎉 
Update the list with multiple participants. 

Despite being the official sponsor of Hacktoberfest, Illa Cloud also has its own separate hacktoberfest event going on which does give physical swag. For now I added it under **I** but in my opinion it should be at the top of the list along with DO, as its one of the official sponsors. 

Thank you.

### CODE REVIEW

File: docs/list.md [hunk 1]
```diff
@@ -26,6 +26,89 @@ See [**Contributing.md**](./contributing.md) to see how to format your pull requ
...
 
+### A
+
+#### **Alby**
+
+- **Swag**: Stickers, Socks, Gadgetst
+- **Requirements**: Be the top 3 contributors with most accepts PRs/MRs.
+- **How to sign up**: [Hacktoberfest Website](https://hacktoberfest.com).
+- **Issues**: All the [open issues]( https://github.com/search?q=org%3AgetAlby+label%3Ahacktoberfest&type=issues&ref=blog.getalby.com).
+- **Notes**: Read the [announcement post](Read more here
```
[the remark this answers is not in the document]
Comment (vimode) [reply docs/list.md:37]: @crweiner Sorry but I am unsure of what am I supposed to do here. 
There are blank lines surrounding this listing, isn't it?

[the remark this answers is not in the document]
Comment (crweiner) [reply docs/list.md:47]: No worries, @vimode! The issue is that anything in the `( )` for a link has to be a URL. The "Read more here" text isn't a URL, so the Markdown linter flagged it. 🎏  Happy to fix this, and thank you for adding so many new companies to the List!

[the remark this answers is not in the document]
Comment (vimode) [reply docs/list.md]: Oh I see now, the text and formatting is all a wrong for the link. Thank you for fixing it.

**Comment by vimode** [comment]
Hey @crweiner since you haven't merged this one yet, can I add some more to the list instead of creating a new PR?

**Comment by crweiner** [comment]
Hi @vimode! Thank you so very much for your massive contribution to the Hacktoberfest Swag List for 2023! I'm so happy to see you back! I greatly appreciate your time and effort in scouting out what companies are participating in Hacktoberfest for another year!!! 🎉 


> Hey @crweiner since you haven't merged this one yet, can I add some more to the list instead of creating a new PR?

I'd like to get this merged now so that way the list is more filled out for today (the first day of Hacktoberfest). Please do submit a new PR if you have more companies to add!

Thank you again, and you rock!!

you rock

**Comment by crweiner** [comment]
I realize I never answered this, @vimode:

> Despite being the official sponsor of Hacktoberfest, Illa Cloud also has its own separate hacktoberfest event going on which does give physical swag. For now I added it under I but in my opinion it should be at the top of the list along with DO, as its one of the official sponsors.

Thanks for calling that out!I know that Appwrite is having their own Hacktoberfest swag and events as well, and they are also an official sponsor of the entire event again this year. 

My thought is to add Illa Cloud and Appwrite listings to the alphabetical section as normal for now, and maybe that's something we may adjust later.

**Comment by vimode** [comment]
> My thought is to add Illa Cloud and Appwrite listings to the alphabetical section as normal for now, and maybe that's something we may adjust later.

Thank you @crweiner for addressing that! I agree the sections might look out of place.

Maybe to highlight their (Appwrite & Illa Cloud) events, we could add a mention of it somewhere in *The Original* listing or just add the anchor link to their names in the title itself? 
I think this could be a good balance between highlighting their events and not breaking the current established hierarchy of the list. 

Just an idea to mull over.

**Pull request merged**